### DISCUSSION TITLE
How To Broadcast Streamers/DJs  When Auto DJ is Running

### DISCUSSION THREAD

**Discussion by hitzgh** [AzuraCast/AzuraCast#6261] [Q&A] [answered]
Hello @BusterNeece Thank you for such a marvelous software, i would like to ask how can i connect to streaming with Streamer/DJ account when auto DJ is running .

i have setup everything but i do no know the details to put in Samcast to be able to connect to the server; i used the dJ Account name and password and connected to the port i was given but i get error (10060)
![image](https://user-images.githubusercontent.com/45140361/236674317-544ace79-b635-4b68-9559-0c9122b4d0aa.png)

**Comment by Piratenkanon** [comment]
is the port correct? normally listen-port +5
and if it's an Icecast-Server you must insert the moint-point

**Comment by LordHelmchen666** [comment, marked as answer]
And check if Port is really open. Not behinde a Firewall.

**Reply by hitzgh** [reply]
is true the port was not opening
